app/translations/src/TranslationFragment.js

```diff
@@ -25,12 +25,14 @@ export default class TranslationFragment extends React.Component<{|
     if (this.props.children === undefined) {
       throw new Error("'TranslationFragment' cannot be used without children.");
     }
-
-    return React.Children.map(this.props.children, child =>
-      React.cloneElement(child, {
-        ...this.props,
-        children: null,
-      }),
-    );
+    return React.Children.map(this.props.children, child => {
+      if (React.isValidElement(child)) {
+        return React.cloneElement(child, {
+          ...this.props,
+          children: null,
+        });
+      }
+      return null;
```

Comment:
Sorry, I didn't realize one thing. Maybe we should return the children here? What about plain string for example? It's not a valid element but it's valid children. This would remove it. Try `<TranslationFragment>just a string</TranslationFragment>`. If I am correct please add a test for it.